samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/artifacts_pulling/ArtifactsPullingTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.kaspersky.kaspressample.artifacts_pulling
+
+import android.Manifest
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.rule.GrantPermissionRule
+import com.kaspersky.kaspressample.MainActivity
+import com.kaspersky.kaspressample.R
+import com.kaspersky.kaspressample.screen.MainScreen
+import com.kaspersky.kaspressample.screen.SimpleScreen
+import com.kaspersky.kaspressample.simple_tests.CheckEditScenario
+import com.kaspersky.kaspresso.kaspresso.Kaspresso
+import com.kaspersky.kaspresso.params.ArtifactsPullParams
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.Rule
+import org.junit.Test
+
+class ArtifactsPullingTest : TestCase(kaspressoBuilder = Kaspresso.Builder.simple {
+    artifactsPullParams = ArtifactsPullParams(enabled = true, artifactsRegex = Regex("screenshots"))
```

Comment:
enabled is a meaningful parameter? Is there any scenario where the user can pass false?